settings.gradle.kts

```diff
@@ -1,5 +1,57 @@
 rootProject.name = "identity-hub"
 
+
+// this is needed to have access to snapshot builds of plugins
+pluginManagement {
+    repositories {
+        maven {
+            url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
+        }
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+dependencyResolutionManagement {
+    repositories {
+        maven {
+            url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
+        }
+        mavenCentral()
+        mavenLocal()
+    }
+    versionCatalogs {
+        create("libs") {
+            from("org.eclipse.edc:edc-versions:0.0.1-SNAPSHOT")
+            version("picocli", "4.6.3")
```

Comment:
Not possible to have these lib versions in the gradle.properties file?

Reply:
I tried on top level of the file and it didn't work. Checking it now probably it would work inside the single block

Reply:
In case i will raise an issue for fixing this

Reply:
I though this was merged, it's still opened so i will change it here